From a68707ff4af5beaeb84ddc3486f3c02eb6368248 Mon Sep 17 00:00:00 2001
From: Xavier Roche <xavrock.os@gmail.com>
Date: Wed, 18 Feb 2015 12:16:31 +0100
Subject: [PATCH] add-PA_PROP_MEDIA_POLICY-Define

Signed-off-by: Xavier Roche <xavrock.os@gmail.com>
---
 avsys-audio-pasimple.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/avsys-audio-pasimple.c b/avsys-audio-pasimple.c
index aeeb999..a56d8bc 100644
--- a/avsys-audio-pasimple.c
+++ b/avsys-audio-pasimple.c
@@ -39,6 +39,8 @@
 #define PA_SIMPLE_PERIOD_TIME_FOR_MID_LATENCY_MSEC			50
 #define PA_SIMPLE_PERIOD_TIME_FOR_HIGH_LATENCY_MSEC			75
 
+#define PA_PROP_MEDIA_POLICY    "media.policy"
+
 #define MSEC_TO_SAMPLE(samplerate,period_time)		(samplerate*period_time/1000)
 
 #define CHECK_VALID_HANDLE(handle) \
-- 
1.9.1

